frontend/src/app/data-entry/source/source.component.html

```diff
@@ -0,0 +1,254 @@
+                                Letter from Radegund
+                            </h5>
+                            <p>A letter written by Radegund describing her vision</p>
+                            <div class="btn-group">
+                                <button class="btn btn-primary">
+                                    <fa-icon [icon]="icons.edit"></fa-icon>
+                                    Edit
+                                </button>
+                                <button class="btn btn-danger">
+                                    <fa-icon [icon]="icons.delete"></fa-icon>
+                                    Delete
+                                </button>
+                            </div>
+                        </div>
+                        <div class="list-group-item">
+                            <h5 class="mb-1">
+                                <fa-icon [icon]="icons.object"></fa-icon>
+                                Hairshirt
+                            </h5>
+                            <p>A penitent gift</p>
+                            <div class="btn-group">
+                                <button class="btn btn-primary">
+                                    <fa-icon [icon]="icons.edit"></fa-icon>
+                                    Edit
+                                </button>
+                                <button class="btn btn-danger">
+                                    <fa-icon [icon]="icons.delete"></fa-icon>
+                                    Delete
+                                </button>
+                            </div>
+                        </div>
+                        <div class="list-group-item">
+                            <h5 class="mb-1">
+                                <fa-icon [icon]="icons.letter"></fa-icon>
+                                Letter from Heaven
+                            </h5>
+                            <p>A letter delivered in a vision</p>
+                            <div class="btn-group">
+                                <button class="btn btn-primary">
+                                    <fa-icon [icon]="icons.edit"></fa-icon>
+                                    Edit
+                                </button>
+                                <button class="btn btn-danger">
+                                    <fa-icon [icon]="icons.delete"></fa-icon>
+                                    Delete
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="btn-group mb-4">
+                        <button class="btn btn-success">
+                            <fa-icon [icon]="icons.create"></fa-icon>
+                            Add a new object
+                        </button>
+                    </div>
+                </ng-template>
+            </div>
+        </div>
+    </div>
+    <div ngbAccordionItem>
```

Comment:
The 'Actions' are now listed in the same accordion block as the agents, objects and places, while the researchers seemed to suggest they wanted to link these in actions and events/episodes at a later stage.

We could create a second accordion, or we could just mention/ask this in the meeting, as this is a quick mock-up anyway.

Reply:
Yes, I wasn't sure about this. There is an "implied order" in the accordion, where actions are the final step, but we could divide it into the two stages that were proposed.

(Btw, I also used the accordion because it came straight out of the component library. It's quite possible we'll end up using something else.)